app/models/manageiq/providers/ibm_cloud/inventory.rb

```diff
@@ -3,7 +3,7 @@ class ManageIQ::Providers::IbmCloud::Inventory < ManageIQ::Providers::Inventory
   require_nested :Parser
   require_nested :Persister
 
-  def self.parsed_manager_name(target)
+  def self.parsed_manager_name(_ems, target)
```

Comment:
This is to accommodate for a recent change to the parent `parsed_manager_name` method which was breaking the tests